bavi/bot.py

```diff
@@ -8,7 +8,7 @@
 from .module_loader import load_modules
 
 log = logging.getLogger('bavi')
-log.setLevel(logging.INFO)
+log.setLevel(logging.DEBUG)
```

Comment:
Should default to `INFO`, we can either have a config key for this or an environment variable.